### PR TITLE
Adding instructions for Youtube Videos.

### PR DESCRIPTION
**Pull request by nicolasgarnier**
But let's only merge once we have the previewer re-deployed and the publisher deployed with the custom elements. @shawnbuso to give us the green light on this.

### CODE REVIEW

File: FORMAT-GUIDE.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ You can also use this to target specific events, for instance:  \
 
     Inline images in your codelab should just work seamlessly. You can re-size them in your codelab document and that width will be applied as a **max-width** on the image in the codelab markup so that images are the same size relative to the text but also scale down appropriately for smaller browsers. 
 
+1. Youtube Videos
+
+    Youtube Videos can be embedded by doing:
+     - Add an image in the document. The image can be a screenshot of the video for instance but it really doesn't really matter since it won't be displayed but replaced by the embedded video.
```
Comment (marcacohen) [inline FORMAT-GUIDE.md]: Two reallys are better than one?  :)

File: FORMAT-GUIDE.md [hunk 2]
```diff
@@ -55,6 +55,13 @@ You can also use this to target specific events, for instance:  \
...
+     - Add an "Alt Text" to the image by doing **Cmd+Opt+Y** or **Right click > "Alt Text..."**
+     - Put a Youtube video link in the **Description** field of the Alt Text. in the format `https://www.yoututbe.com/watch?v=[video_ID]`
```
Comment (marcacohen) [inline FORMAT-GUIDE.md]: Can you include a start time?  Would be good to state whether that's supported or not.

Comment (nicolasgarnier) [reply FORMAT-GUIDE.md]: Currently it's not. But let's file a feature request. I can add that in another PR.

File: FORMAT-GUIDE.md (hunk 1 above)
Comment (nicolasgarnier) [reply FORMAT-GUIDE.md]: Fixed! 🤣

File: FORMAT-GUIDE.md (hunk 2 above)
Comment (marcacohen) [reply FORMAT-GUIDE.md]: Can you state here that it's not currently supported (so it's clear)?

Comment (marcacohen) [reply FORMAT-GUIDE.md]: I realize the set of things not supported is infinite :) but this is an obvious one I think people will wonder about.

Comment (nicolasgarnier) [reply FORMAT-GUIDE.md]: Done!

Comment (marcacohen) [reply FORMAT-GUIDE.md]: Thanks!

**Pull request merged**